media/css/mozorg/home/home-b.scss

```diff
@@ -0,0 +1,576 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+@import '../../hubs/buttons';
+
+body {
+    @include open-sans;
```

Comment:
I totally welcome using `open-sans` for body copy here :) (I'm hoping we can switch the rest of the new pages to do this by default later)